examples/01b-Getting-started-Pytorch.ipynb

```diff
@@ -0,0 +1,487 @@
+{
```

Comment:
<p>We do not calculate it anymore, do we?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/dataloader/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>yes, thank you for spotting it, my bad for not removing it, removed it now</p>